### PR TITLE
modified main.rb and added a new file methods

### PR DESCRIPTION
**Pull request by Sick-Carlito**

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require_relative '../lib/methods.rb'
+
+class UserInterface
+  def initialize
+    @game = Game.new
```
Comment (stratospherique) [inline bin/main.rb]: - `Game` here is supposedly the **Methods** class that you have in the `lib` folder

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    spot = @game.board_index(spot)
+    if @game.valid_move?(spot)
+      @game.set_input(spot, @game.current_player)
+    else
+      puts 'This spot hasbeen taken or you inputed a wrong value'
+      puts 'Please enter a valid option between 1-9'
+      display_board
+      turn
+    end
+    display_board
```
Comment (stratospherique) [inline bin/main.rb]: - Displaying the board twice might be unpleasant from a UX perspective.

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Impressive work Team :clap: However, I have some notes that you need to address in order to get approval for this milestone. check :arrow_down: 

- **[REQUIRED]** The project is collaborative. Why I can only see @Sick-Carlito pushed commits? :pray:  it's mandatory to explain.

- **[REQUIRED]** Add more info to Readme.md and make it more descriptive.

- I added some code comments below. Take a look at them and refactor your code.

### That's it :relaxed: When finished, kindly resubmit another code review request :pray: 

_Happy coding_ :v: 
**Ahmed Mahfoudh**

File: bin/main.rb (hunk 1 above)
Comment (Sick-Carlito) [reply bin/main.rb]: My partner wasn't around for a while but he's back now. You will be able to see his commits soon enough.

**Review by AdnanAfsari — APPROVED** [review]
**Status : Approved**

Hello @Sick-Carlito @cochabambinoski ✋ 

You made all the changes reviewer requested. so its a 👍 from me.

Happy Coding
Adnan Afsari

**Comment by Sick-Carlito** [comment]
Thanks guys!!

**Pull request merged**